src/View/Widget/FileWidget.php

```diff
@@ -65,7 +65,7 @@ public function render(array $data, ContextInterface $context)
             'templateVars' => $data['templateVars'],
             'attrs' => $this->_templates->formatAttributes(
                 $data,
-                ['name', 'val']
+                ['name']
```

Comment:
`$data['val']` is already unset few lines above, so including it in exclude list is not necessary.